repo/maintenancestats/types.go

```diff
@@ -0,0 +1,28 @@
+// Package maintenancestats manages statistics for maintenance tasks.
+package maintenancestats
+
+import (
+	"encoding/json"
+)
+
+// RawStats holds the raw data for maintenance statistics.
+type RawStats struct {
+	Kind string          `json:"kind,omitempty"`
+	Raw  json.RawMessage `json:"raw,omitempty"`
+}
+
+// Stats defines the methods for maintenance statistics.
+type Stats interface {
```

Comment:
Either `Summarizer` or `ExtraSumarizer`
```suggestion
type Summarizer interface {
```